mozilla/blurts-server
# MNTOR-2103: log activation/deactivation errors to sentry

# References: 
Jira: MNTOR-2103




# Description
Legitimate 400 errors coming from OneRep should be logged in sentry and 200 should be returned to event broker to prevent spamming


# Screenshot (if applicable)

![Screenshot 2023-08-30 at 6 42 43 PM](https://github.com/mozilla/blurts-server/assets/42494162/b9d14630-9b4c-4cda-9103-8304bf6c74c5)


# How to test
In heroku, notice that the existing errors no longer exist, and corresponding error is logged in Sentry


# Checklist (Definition of Done)
